settings/common.py

```diff
@@ -144,3 +146,5 @@
 }
 
 # ALLAUTH SETTINGS
+
+CORS_ORIGIN_ALLOW_ALL = True
```

Comment:
Lets have a limited set of allowed hostnames.

1. localhost
2. 127.0.0.1
3. ec2-54-242-225-49.compute-1.amazonaws.com